platform/ios/src/MLNMapView.h

```diff
@@ -1253,6 +1253,25 @@ MLN_EXPORT
  */
 - (void)flyToCamera:(MLNMapCamera *)camera withDuration:(NSTimeInterval)duration peakAltitude:(CLLocationDistance)peakAltitude completionHandler:(nullable void (^)(void))completion;
 
+
+/**
+ Moves the viewpoint to a different location using a transition animation that
+ evokes powered flight and an optional transition duration and peak altitude.
```

Comment:
This method does take a duration (and edge padding), but unlike the method above, it doesn’t take a peak altitude.

> I'm not sure if this makes sense since `MLNMapCamera` can have a `altitude`/`viewingDistance`.

The peak altitude is the altitude you want the camera to have if you get the map’s camera at the moment when the map is zoomed out the most.

Reply:
I updated the documentation as required.